www-client/firefox/firefox-128.0.3.ebuild

```diff
@@ -221,7 +217,8 @@ llvm_check_deps() {
 
 		if use pgo ; then
 			if ! has_version -b "=sys-libs/compiler-rt-sanitizers-${LLVM_SLOT}*[profile]" ; then
-				einfo "=sys-libs/compiler-rt-sanitizers-${LLVM_SLOT}*[profile] is missing! Cannot use LLVM slot ${LLVM_SLOT} ..." >&2
+				einfo "=sys-libs/compiler-rt-sanitizers-${LLVM_SLOT}*[profile] is missing!" >&2
+				einfo "Cannot use LLVM slot ${LLVM_SLOT} ..." >&2
```

Comment:
_Hello @fkobi,_
I'm just curious, why not to use `ewarn` here?

Reply:
It's an informative message, not a fatal one. *llvm.eclass* works in such a way that it tries to find a suitable implementation. Say it tried 18 and if that was incomplete, it'd roll to the next one (17, 16, and so on). These are **probably** useless after migration to *llvm-r1.eclass.*